### PR TITLE
EXLM-55 MD converter for video

### PR DESCRIPTION
**Pull request by ruhisingh1**

### CODE REVIEW

**Comment by ahmed-musallam** [comment]
very small 2 changes requested, otherwise LGTM! feel free to merge

**Review by ahmed-musallam — CHANGES_REQUESTED** [review]

File: src/modules/ExlMd2Html.js [hunk 1]
```diff
@@ -74,10 +76,17 @@ function converter (mdString, nested = false) {
   raw(hast);
   rehypeFormat()(hast);
 
-  return toHtml(hast, {
+  const htmlForDomTrasnsformation =  toHtml(hast, {
```
[the remark this answers is not in the document]
Comment (ruhisingh1) [reply src/modules/ExlMd2Html.js]: Fixed

File: src/modules/blocks/create-video.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+		
+        const div = document.createElement('div');
+        const img = document.createElement('img');
+        const href = element.querySelector('iframe').src;
+
+        const p1 = document.createElement('p');
+        p1.append(img);
+
+        const p2 = document.createElement('p');
+        p2.innerHTML = '<a href="' + href + '">' + href + '</a>';
```
[the remark this answers is not in the document]
Comment (ruhisingh1) [reply src/modules/blocks/create-video.js]: Fixed

**Review by ruhisingh1 — COMMENTED** [review]
Addressed review comments.

**Comment by ahmed-musallam** [comment]
LGTM. please merge

**Review by ahmed-musallam — APPROVED** [review]

**Review by tprasad05 — APPROVED** [review]

**Pull request merged**